website/www/site/content/en/get-started/downloads.md

```diff
@@ -88,6 +88,14 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.38.0 (2022-04-20)
+Official [source code download](https://www.apache.org/dyn/closer.cgi/beam/2.38.0/apache-beam-2.38.0-source-release.zip).
+[SHA-512](https://downloads.apache.org/beam/2.38.0/apache-beam-2.38.0-source-release.zip.sha512).
+[signature](https://downloads.apache.org/beam/2.38.0/apache-beam-2.38.0-source-release.zip.asc).
+
+[Release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12319527&version=12351169)
+[Blog post](/blog/beam-2.38.0).
+
```

Comment:
We can do this in a follow up - but I just remembered that previously the release guide said you should update the last release to point to archive.apache.org. When I released 2.37.0 I wasn't sure why that went away, but I didn't check on it.  I just checked now and https://downloads.apache.org/beam/ only has the latest (2.37.0) release. So it seems this step is necessary.

I can send a PR to fix the other links and add this back to the release guide.

Reply:
Actually would you mind updating the 2.37.0 links here, as in https://github.com/apache/beam/pull/11727?

The problem seems to be that the release guide switched to an example PR that doesn't do this (https://github.com/apache/beam/pull/15068)

Reply:
That and the instructions for it are under PMC-only finalization instead of being under a section on the Beam website PR itself. I switched all the links to the archive URL.

Reply:
Cool - I can also follow-up with a change to clarify the release guide (at the very least linking to this PR as an example).